providers/ibm/ibm_provider.go

```diff
@@ -39,7 +39,7 @@ func (p *IBMProvider) Init(args []string) error {
 	p.VPC = args[3]
 
 	var err error
-	if p.Region != NoRegion {
+	if p.Region != DefaultRegion && p.Region != NoRegion {
```

Comment:
?? Why changes in ibm provider?

Reply:
It was unintended changed introducing during the manual merge with upstream.
I've rebase and squashed the logs for simplicity